include/liblightnvm_cli.h

```diff
@@ -128,6 +128,7 @@ enum nvm_cli_opt_type {
 	NVM_CLI_OPT_FILE_OUTPUT = 1 << 5,
 	NVM_CLI_OPT_VAL_DEC = 1 << 6,
 	NVM_CLI_OPT_VAL_HEX = 1 << 7,
+    NVM_CLI_OPT_TERSE = 1 << 8,
```

Comment:
Good +1

Reply:
btw. double-check that the indentation matches the other values of the enum.

Reply:
Fixed indentation.